draft-mvp/src/Player.js

```diff
@@ -1,12 +1,48 @@
 import React, { Component } from "react";
+import Modal from "./Modal.js";
 
 export default class Player extends Component {
   constructor(props) {
     super(props);
+
+    this.state = {
+      showModal: false,
+      drag: false
+    };
   }
 
+  handleModal = () => {
+    console.log(`handlemodal ${this.props.player}`);
+    this.setState({
+      showModal: true
+    });
+  };
+
+  handleCloseModal = () => {
+    console.log("close modal");
+    this.setState({
+      showModal: false
+    });
+  };
```

Comment:
This block looks good. I'm wondering if this code should be contained in the modal component directly or not.